storages/customers_storage.py

```diff
@@ -0,0 +1,73 @@
+from config.db_conn import conn
+from psycopg2 import sql, Error
+from datetime import datetime
+from exceptions.customer_exceptions import EntityNotFound
+
+
+class CustomerStorage:
+    def __init__(self):
+        self.customers = []
+        self.db = conn
+
+    def update_customer(self, customer_update) -> tuple:
+        try:
+            with self.db.cursor() as cursor:
+                query = sql.SQL(
+                    """
+                                UPDATE customers
+                                SET name =%s, email =%s, updated_at=%s
+                                WHERE id = %s
```

Comment:
@hesfra  Falta um "WHERE id = %s AND active = true;" pq se o active for false, o arquivo foi "deletado", então não seria possível editar, certo @Arthuralentejo  @AurelioNM  ?